packages/cli-kit/src/node/node-package-manager.ts

```diff
@@ -204,8 +256,18 @@ interface AddNPMDependenciesIfNeededOptions {
   signal?: AbortSignal
 }
 
+/**
+ * An interface that represents a dependency name with its version
+ */
 export interface DependencyVersion {
+  /**
+   * Dependency name
+   */
   name: string
+
+  /**
+   * Dependency version
+   */
```

Comment:
This is something we should discuss as a team, but I think this kind of comments don't add much value... I'm talking about this specific example, I find it useful for other cases.